app/views/components/_email-signup-section.html.erb

```diff
@@ -0,0 +1,45 @@
+<%
+  details ||= {}
+  link_text ||= false
+  link_href ||= false
+  intro ||= false
+  full_width ||= false
+  track_action ||= false
+  track_label ||= false
+  transparent_background ||= false
+  css_classes ||= false
+  classes ||= []
+  tracking_data = track_action ? {
+    module: "track-click",
+    "track-category": "emailAlertLinkClicked",
+    "track-action": track_action,
+    "track-label": track_label
+  } : {}
+
+  classes << css_classes if css_classes
+  classes << "app-c-email-signup--full-width" if full_width
+  classes << "app-c-email-signup--no-intro" if !intro
+  classes << "app-c-email-signup--transparent_background" if transparent_background
+
+  #  if full_width is true, wrap content with govuk layout classes for full width styling
+  def full_width_wrapper(is_full_width, &block)
```

Comment:
What's the requirement for this? Where is it used?

Reply:
Having looked into this more I can see it's for the desktop version. I've consulted with design and got a new version attached to the card. Basically I think we can strip all of this out and use the transparent background version of the component inside the existing grey box (the box that already wraps the action links on the hub pages).

Reply:
I've reworked/simplified how the whole thing works. The design tweak makes it much simpler.